### PR TITLE
fix: Blackduck performance threshold

### PR DESCRIPTION
**Pull request by tomfrenken**
This PR switches our Blackduck scans from a per-branch scan to nightly scans, to avoid hitting the performance threshold.

Closes SAP/cloud-sdk-backlog#574.

### CODE REVIEW

File: .github/workflows/blackduck.yml [hunk 1]
```diff
@@ -1,9 +1,8 @@
 name: blackduck-scan
 
 on:
-  pull_request: ~
-  push:
-    branches: ['main']
+  schedule:
```
Comment (jjtang1985) [inline .github/workflows/blackduck.yml:4]: Maybe we can add a manual trigger so we can make a scan before a release.

Comment (jjtang1985) [reply .github/workflows/blackduck.yml:4]: You can find the trigger in the release pipeline.

Comment (tomfrenken) [reply .github/workflows/blackduck.yml:4]: Ah good idea.

Comment (tomfrenken) [reply .github/workflows/blackduck.yml:5]: ✅

**Review by florian-richter — APPROVED** [review]
LGTM

**Pull request merged**